guides/release/components-update/component-state.md

```diff
@@ -0,0 +1,306 @@
+  @tracked count = 0;
+
+  @action
+  change(amount) {
+    this.count = this.count + amount;
+  }
+  @action
+  increment() {
+    this.count = this.count + 1;
+  }
+
+  @action
+  decrement() {
+    this.count = this.count - 1;
+  }
+}
+```
+
+Next, we'll update the template to turn the click handler into a function that passes the amount as an argument.
+
+```handlebars {data-filename="app/components/counter.hbs" data-diff="-3,+4,-5,+6"}
+<p>{{this.count}}</p>
+
+<button {{on "click" this.increment}}>+1</button>
+<button {{on "click" (fn this.change 1)}}>+1</button>
+<button {{on "click" this.decrement}}>-1</button>
+<button {{on "click" (fn this.change -1)}}>-1</button>
+```
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        An event handler takes a function as its second argument. When there are no arguments to the
+        function, you can pass it directly, just like in JavaScript. Otherwise, you can build a
+        function inline by using the <code>fn</code> syntax.        
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="">
+  </div>
+</div>
+
+## Computed Values
+
+Let's say we want to add a button to our counter that allows us to double the current count. Every time we press the button, the current count doubles.
+
+Based on what we've already learned, we'll need:
+
+- A piece of state that represents the number to multiply the `count` by (the `multiple`)
+- An action to double the `multiple`
+- A button in the template that calls the action
+
+But we'll also need a way to multiply the `count` by the `multiple` and show it in the template.
+
+Let's start with what we know already. Add the `multiple` tracked property and an action called `double` that doubles the `multiple`.
+
+```js {data-filename="app/components/counter.js" data-diff="+7,+9,+10,+11,+12"}
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+
+export default class extends Component {
+  @tracked count = 0;
+  @tracked multiple = 1;
+
+  @action
+  double() {
+    this.multiple = this.multiple * 2;
+  }
+
+  @action
+  change(amount) {
+    this.count = this.count + amount;
+  }
+}
+```
+
+And update the template to call the `double` action. We'll also add `this.multiple` to our output to confirm that the button is working.
+
+```handlebars {data-filename="app/components/counter.hbs" data-diff="+2,+6"}
+<p>{{this.count}}</p>
+<p>× {{this.multiple}}</p>
+
+<button {{on "click" (fn this.change 1)}}>+1</button>
+<button {{on "click" (fn this.change -1)}}>-1</button>
+<button {{on "click" this.double}}>Double It</button>
+```
+
+To get the multiplied number into the template, we'll use a normal [JavaScript getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get).
```

Comment:
```suggestion
To get the multiplied number into the template, we'll use a [JavaScript getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get).
```

A "normal" JavaScript getter here makes me think we're going to use an abnormal one later. Can probably just cut that here.